packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/Settings.tsx

```diff
@@ -29,9 +31,10 @@ function Settings({ data }: Props) {
     control,
     register,
     formState: { errors },
-  } = useFormContext<Application>();
+  } = useFormContext<ApplicationForm>();
 
   const { type: applicationType } = data;
+  const { field: customData } = useController({ name: 'customData', control });
```

Comment:
Can we use `Controller` components (align with other fields on this page)?

Reply:
updated